### PR TITLE
CVSB-5615: Adding 2 conditions to supress certificate gen.

### PR DESCRIPTION
**Pull request by bsiddappa**

### CODE REVIEW

**Comment by KHodderDVSA** [comment]
Where are the tests for this?

**Comment by bsiddappa** [comment]
> Tests?

I see that certGen.ts function has been skipped from sonar and not unit test for the same. Guess will have to remove it from the exclusion and write a fresh unit test for the class. Will work on this.

**Comment by bsiddappa** [comment]
Added tests for certGen.ts function and its coverage stands to 78.8% as of now. There is another condition where we check for testId format which is not covered in this test as its output was changed to console error through another ticket(now merged with develop) and this branch still has old code(which throws 500 causing the message to remain in queue).
Kindly review.

File: tests/integration/certGen.intTest.ts [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+import {CertificateUploadService} from "../../src/services/CertificateUploadService";
+import { certGen } from "../../src/functions/certGen";
+import lambdaTester from "lambda-tester";
+import sinon from "sinon";
+import {log, error} from "console";
+
+describe("Invoke certGen Function", () => {
+    context("when the certGen function is invoked with valid test result", () => {
+        const lambda = lambdaTester(certGen);
+        const payload: any =
```
Comment (KHodderDVSA) [inline tests/integration/certGen.intTest.ts]: Could pull this out into a file and modify as required, but not essential

**Pull request merged**